BazelExtensions/xcodeproject.bzl

```diff
@@ -188,6 +196,40 @@ def _install_xcode_project_impl(ctx):
         'echo "' + output_proj + '" > ' + ctx.outputs.out.path,
     ]
 
+    if ctx.attr.bazel_build_service_config:
```

Comment:
A simple improvement we consider - namespacing these by the project name for a sensible default - otherwise LGTM

Reply:
Done